docs/content/language-cheatsheet.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Language Cheatsheet
+navtitle: Language Cheatsheet
+kind: documentation
+weight: 5
+toc: true
+---
+
+## Assignment and Equality
+
+**Outside Rules**
+
+```ruby
+# assign variable allowed to the set "hooli.com" and "initech.com"
+allowed = {"hooli.com", "initech.com"}
+```
+
+**Inside Rules**
```

Comment:
The section names `Outside Rules` and `Inside Rules` are bit unclear.

Reply:
See last comment.